src/commands/zrank.json

```diff
@@ -4,7 +4,7 @@
         "complexity": "O(log(N))",
         "group": "sorted_set",
         "since": "2.0.0",
-        "arity": 3,
+        "arity": -3,
         "function": "zrankCommand",
```

Comment:
oops... need add history about the `withscore` argument like:

```
        "history": [
            [
                "7.2.0",
                "Added the `withscore` argument."
            ]
        ],
```

Reply:
Totally forgot...just added.